packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/ZoomInButton/ZoomInButtonContainer.js

```diff
@@ -17,10 +17,31 @@ function storeMapper (stores) {
 @inject(storeMapper)
 @observer
 class ZoomInButtonContainer extends React.Component {
-  render () {
+  constructor () {
+    super()
+    this.onPointerDown = this.onPointerDown.bind(this)
+    this.onPointerUp = this.onPointerUp.bind(this)
+    this.timer = ''
+  }
+
+  onPointerDown (event) {
     const { zoomIn } = this.props
+    this.timer = setInterval(zoomIn, 100)
```

Comment:
It's possible for really fast clicks on the Zoom In button to not register a _zoom in action._

This is because, if the `onPointerUp () { clearInterval(this.timer) }` occurs in less than 100ms, then the `zoomIn()` action isn't even triggered once.

One solution would be...

```
zoomIn()
this.timer = setInterval(zoomIn, 100)
```

(On my MacBook, tapping quickly on the trackpad can be done in less than 100ms, so no zoom in action occurs. Physically clicking down on the trackpad is usually done in more than 100ms, so in most cases, a zoom in action occurs.)